egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -208,11 +215,13 @@ def process_args(args):
                 args.deriv_truncate_margin))
 
     if (not os.path.exists(args.dir)
-            or not os.path.exists(args.dir+"/configs")):
-        raise Exception("This scripts expects {0} to exist and have a configs "
+            or (not os.path.exists(args.dir+"/configs") and
+                not os.path.exists(args.input_model))):
+        raise Exception("This scripts expects {0} to exist. Also either of --trainer.input-model "
```

Comment:
also this sentence does not make sense. Maybe "Either the --trainer.input-model option should be supplied or ....>"
and check line length.